staging/object-bucket-claim/templates/object-bucket-claim.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: {{ template "object-bucket-claim.fullname" . }}
+  labels: {{ include "object-bucket-claim.labels" . | nindent 4 }}
+spec:
+  # To create a new bucket specify either `bucketName` or
+  # `generateBucketName` here. Both cannot be used. To access
+  # an existing bucket the bucket name needs to be defined in
+  # the StorageClass referenced here, and both `bucketName` and
+  # `generateBucketName` must be omitted in the OBC.
+  bucketName: {{ .Values.bucketName }}
+  storageClassName: {{ .Values.storageClassName }}
+  additionalConfig:
```

Comment:
Can we just use `.Values.additionalConfig` to populate the entire block?
something like
```
{{- if .Values.additionalConfig }}
  additionalConfig:
{{ toYaml .Values.additionalConfig | indent 4 }}
{{- end }}
```

Reply:
done